google/src/main/scala/org/broadinstitute/dsde/rawls/google/GoogleUtilities.scala

```diff
@@ -61,20 +63,22 @@ trait GoogleUtilities extends LazyLogging with Retry {
     }
   }
 
-  protected def executeGoogleCall[A,B](request: AbstractGoogleClientRequest[A])(processResponse: (com.google.api.client.http.HttpResponse) => B): B = {
+  protected def executeGoogleCall[A,B](request: AbstractGoogleClientRequest[A])(processResponse: (com.google.api.client.http.HttpResponse) => B)(implicit counters: GoogleCounters): B = {
     val start = System.currentTimeMillis()
     Try {
       request.executeUnparsed()
     } match {
       case Success(response) =>
         logGoogleRequest(request, start, response)
+        instrumentGoogleRequest(request, start, Right(response))
         try {
           processResponse(response)
         } finally {
           response.disconnect()
         }
       case Failure(httpRegrets: HttpResponseException) =>
         logGoogleRequest(request, start, httpRegrets)
+        instrumentGoogleRequest(request, start, Left(httpRegrets))
         throw httpRegrets
       case Failure(regrets) =>
         logGoogleRequest(request, start, regrets)
```

Comment:
ok. Only thing is we won't have a response status code (since this is _not_ a `HttpResponseException`) but we could instrument with an unknown status.